tests/Feature/MemberControllerTest.php

```diff
@@ -0,0 +1,875 @@
+<?php
```

Comment:
All the tests should also check the JSON response and the indentation is broken in many places. Please run `phpcs` and fix the code style `phpcbf` can fix most of them automatically.